aiodocker/utils.py

```diff
@@ -57,3 +60,67 @@ def httpize(d: Optional[dict]) -> Optional[dict]:
     d = decoder.decode(b'', final=True)
     if d:
         yield d
+
+
+def clean_config(config: Optional[dict]) -> dict:
```

Comment:
lets also add tests for this helpers, should be very easy, we can do this in separate PR since, current one is already large.